sklearn/utils/_estimator_html_repr.py

```diff
@@ -144,6 +156,19 @@ def _get_visual_block(estimator):
     )
 
 
+def _get_estimator_doc_url(estimator):
```

Comment:
This is dead code, isn't it?

Reply:
I thought I had pushed that change, but apparently I didn't.. I removed that now.